.pre-commit-config.yaml

```diff
@@ -42,6 +42,25 @@ repos:
       - id: check-yaml
         name: "Check YAML"
 
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args: [--target-version=py39]
```

Comment:
Yeah, that issue's still unresolved and I only set the minimum version, and I don't think I've seen other projects list multiple.

Let's defer to @JelleZijlstra :)

I don't think we need to follow PEP 8 here, that's explicitly for the stdlib, and other https://github.com/python/ projects also use Black. It's so much easier just to run it and not have to worry about formatting. Similarly, I'd prefer to keep the defaults, including line lengths.

Reply:
Its not nearly important enough for me to block the "agreement on this issue" required by PEP 8 to nominally deviate from its stated guidance, "Limit all lines to a maximum of 79 characters." unless its an issue for others, particularly since the code probably wouldn't wrap nearly as neatly and the existing `pep_sphinx_extensions` code doesn't follow it either.

In that case, the suggestion becomes:

```suggestion
        args:
          - '--target-version=py39'
          - '--target-version=py310'
```